packages/grid_client/src/primitives/nodes.ts

```diff
@@ -373,7 +374,7 @@ class Nodes {
       ipv6: options.accessNodeV6,
       gateway: options.gateway,
       certification_type: options.certified ? "Certified" : "",
-      farm_ids: [options.farmId],
+      farm_ids: options.farmIds ? options.farmIds : [options.farmId],
```

Comment:
`farmId` is more specific than `farmIds`, so I think if the `farmId` is provided, we should use it and ignore the `farmIds`
```suggestion
      farm_ids: options.farmId ? [options.farmId] : ptions.farmIds,
```